api/server/router/local/image.go

```diff
@@ -292,9 +294,14 @@ func (s *router) postBuild(ctx context.Context, w http.ResponseWriter, r *http.R
 		// Do not write the error in the http output if it's still empty.
```

Comment:
Well, I noticed that the `errf` function is called in `postBuild` when something goes wrong. Therefore, it looks natural to me to use it for sending the build output and by that saving code lines on `fmt.Fprintf(output...)` at these parts.